kivy/uix/recycleview/__init__.py

```diff
@@ -155,6 +158,94 @@ def build(self):
 Please see the `examples/widgets/recycleview/basic_data.py` file for a more
 complete example.
 
+## Viewclass State
+
+As the viewclass widgets are reused or instantiated as needed by the RecycleView, the order and content of the widgets are mutable. Any state change to a single widget will stay with that widget, even if the data assigned to it from the RecycleView.data dict changes.
+
+Two methods for managing state changes in viewclass widgets:
+
+* Store state in the RecycleView.data Model
+* Generate state changes on-the-fly by catching updates or binding
```

Comment:
```suggestion
Viewclass State
^^^^^^^^^^^^^^^

Because the viewclass widgets are reused or instantiated as needed by the
:class:`RecycleView`, the order and content of the widgets are mutable. So any state
change to a single widget will stay with that widget, even when the data assigned to
it from the :attr:`~RecycleView.data` dict changes, unless :attr:`~RecycleView.data`
tracks those changes or they are manually refreshed when re-used.

There are two methods for managing state changes in viewclass widgets:

1. Store state in the RecycleView.data Model
2. Generate state changes on-the-fly by catching :attr:`~RecycleView.data`
   updates and manually refreshing.
```